ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/controller/UsersController.java

```diff
@@ -15,8 +20,17 @@ public UsersController(LdapService ldapService) {
     }
 
     @PostMapping("/users")
-    public User getUser(@RequestBody UserCredentials userCredentials) {
-        return ldapService.authenticate(userCredentials);
+    public ResponseEntity<User> getUser(@RequestBody UserCredentials userCredentials) {
+        User user = ldapService.authenticate(userCredentials);
+        if (user != null) {
+            return new ResponseEntity<User>(user, HttpStatus.OK);
+        } else {
+            return new ResponseEntity(new EmptyJsonResponse(), HttpStatus.OK);
+        }
+    }
+
+    @JsonSerialize
+    public class EmptyJsonResponse {
```

Comment:
I found a few ways to return an empty JSON object, but all of them are more verbose than expected.